json-guides/retry-timeout.json

```diff
@@ -91,6 +91,64 @@
                 }
             ]
         },
+        {
+            "name": "TimeoutAnnotation",
+            "title": "Adding the @Timeout annotation",
+            "description": [
+                "Now that you've seen the application easily fail when it loads for an indefinite amount of time, let's apply a Timeout policy to to limit the time the request is made to a transaction history and to prevent the page from loading forever.",
+                "With a Timeout policy, the microservice fail with <code>TimeoutException</code> after the set amount of time has passed when the request to a transaction history is made.",
+                "The <code>@Timeout</code> annotation has one parameter that specifies the amount of time it took to make a transaction history request to the service. If the parameter is not specified, the default is <code>1000</code> milliseconds.",
+                "After you modify your <code>server.xml</code> file to <a href='#enabling-microprofile-fault-tolerance'> include the Fault Tolerance feature</a>, add a Timeout policy to the transaction history microservice."
+            ],
+            "instruction": [
+                "Add the <code>@Timeout</code> annotation with a value of <code>2000</code> on line 9, before the <code>showTransactions</code> method, or click <action title='Add @Timeout' onclick=\"retryTimeoutCallback.addTimeoutButton(event, 'TimeoutAnnotation')\">@Timeout(2000)</action>. This annotation limits the time it tooks to retrieve a transaction history to 2000 milliseconds before a <code>TimeoutException</code> occurs. Then, click <action title='Run' onclick=\"retryTimeoutCallback.saveButtonEditor(event, 'TimeoutAnnotation')\"><b>Run</b></action> on the editor menu pane."
```

Comment:
This annotation limits the allowed time for the transaction history request to 2000 milliseconds ...